README.md

```diff
@@ -13,9 +13,10 @@
 
 It supports optional parameters:
 
-- `mode` - enum `bytes` or `rows`, default `bytes`
+- `mode` - enum (`bytes`, `rows`, `slices`), default `bytes`
 - `bytesPerSlice` (`int`) - for `mode = bytes`, maximum size of the one slice in bytes before compression, default `524 288 000` (`500 MiB`)
 - `rowsPerSlice` (`int`) - for `mode = rows`, maximum rows in the one slice, default `1 000 000`
+- `numberOfSlices` (`int`) - for `mode = slices`, fixed number of slices, default `60`
```

Comment:
V povodnom zadani bolo, ze idealne ked CSVcko rozdelime na `60 casti` ... doteraz to nedavalo prakticky zmysel, ... no z testov nad Synapse vyzera, ze Synapse moze pomoct viacero malych casti ... tak to tu pridavam, aby som to zmeral.